### DISCUSSION TITLE
Is this app safe for my laptop (battery issue)

### DISCUSSION THREAD

**Discussion by deveers** [BartoszCichecki/LenovoLegionToolkit#879] [Q&A] [answered]
So I've been using this app on my laptop which I bought about 4 months ago. Before I used the app, I checked the battery capacity and it was upto the design capacity

But today I checked again and it's now at 90% of design and it said 162 cycle count. That was 6 hours ago. I checked again and now it says 164 cycle counts? Is it the fault of the app or is it 'normal'? I've been mainly using it to set the laptop on battery conservation mode constantly and leave it plugged in all the time and I game a lot

My laptop - Ideadpad Gaming 3

**Comment by BartoszCichecki** [comment, marked as answer]
This app has no impact on battery life. All statistics like cycle count are provided by Windows.
